Fix links to app-icons and splash-screens

j-carter opened the pull request:
In the "Configuring your app with app.json" the links to the guide pages for app icons and splash screens are backwards. I just switched them so they'll be correct.

j-carter committed:
Fix links to app-icons and splash-screens

In the "Configuring your app with app.json" the links to the guide pages for app icons and splash screens are backwards.

tsapeta reviewed (changes requested):
Hi @j-carter, thanks for your contribution!
Could you please apply the same change to `docs/pages/versions/unversioned/tutorial/follow-up.md`? This is the version of docs that will be used in future SDK releases, so if we don't put this change there then it will be fixed only in SDK36 😉

j-carter committed:
Fix links to app-icons and splash-screens

The links to the app-icons and splash-screens pages were reversed. I just switched them so they would be correct.

j-carter commented:
Thanks @tsapeta  for the feedback.  I have updated the extra file just now.
